packages/teams-js/src/internal/communication.ts

```diff
@@ -615,39 +683,43 @@ function handleParentMessage(evt: DOMMessageEvent): void {
   if ('id' in evt.data && typeof evt.data.id === 'number') {
     // Call any associated Communication.callbacks
     const message = evt.data as MessageResponse;
```

Comment:
>MessageResponse [](http://example.com/codeflow?start=32&length=15)

Would the data coming back over this pipe actually be a `SerializedMessageResponse`? And we should deserialize it first, before examining its data or manipulating it?

Reply:
I thought about the deserialization, but we don't pass along the `MessageResponse` object anywhere in the code. It is mostly used during either data processing on our end or creation of the object right before sending. I do think that the naming for this object should change to `SerializedMessageResponse` since it is the response received from the host or sent to the host.